_translator-files/po/fr/Installation-for-Linux.po

```diff
@@ -206,12 +206,12 @@ msgstr "## Tout est installé ?"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:64
 msgid "Take a look at the"
-msgstr "Jetez un coup d'oeil à la"
+msgstr "Jetez un coup d'œil à la"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:65
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```

Comment:
Is adding the space between `{` and `:` correct?

Reply:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but id doesn't apply here) and I didn't corrected it. I removed it.